packages/reactotron-server/tailwind.js

```diff
@@ -41,6 +41,9 @@ module.exports = {
   backgroundColors: {
     sectionPicker: colors.black,
     content: colors.greyDarkest,
+    expandButton: colors.greyDarker,
```

Comment:
I like the extra roles... i wonder if we need the `expand` prefix at all though.  Will all buttons be like this?  Not sure. Now I'm thinking outloud.